src/game.cpp

```diff
@@ -7278,6 +7281,25 @@ void game::zones_manager()
     refresh_all();
 }
 
+void game::pre_print_all_tile_info( const tripoint &lp, const catacurses::window &w_info,
+                                    int column, int &first_line, const int last_line,
+                                    bool draw_terrain_indicators, const visibility_variables &cache )
+{
+    if( !column ) {
+        column = 0;
+    }
+    if( !draw_terrain_indicators ) {
+        draw_terrain_indicators = 0;
```

Comment:
`draw_terrain_indicators` is `bool`, why casting it to `int`?

Reply:
i'm changing it to false, sorry about that